### PR TITLE
PR

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: dags/landsat_scenes_sync/landsat_scenes_sync.py [hunk 1]
```diff
@@ -31,18 +31,18 @@
...
-    "version": "0.6.2",
+    "start_date": datetime(2021, 1, 1),
+    "version": "0.7.0",
 }
 # [END default_args]
 
 # [START instantiate_dag]
 dag = DAG(
-    "landsat_scenes_sync",
+    "landsat_scenes_identifying",
```
Comment (pindge) [inline dags/landsat_scenes_sync/landsat_scenes_sync.py:41]: if u are changing this name, shouldnt the file name change too?/

**Review by pindge — CHANGES_REQUESTED** [review]

File: dags/landsat_scenes_sync/landsat_scenes_sync.py (hunk 1 above)
Comment (carioca-au) [reply dags/landsat_scenes_sync/landsat_scenes_sync.py:41]: I reckon you've got a point, I'll change

Comment (carioca-au) [reply dags/landsat_scenes_sync/landsat_scenes_sync.py:41]: Done

**Comment by pindge** [comment]
Can u also update ur readme as well??

**Pull request closed without merging**